rpg/message_box.py

```diff
@@ -4,15 +4,16 @@
 
 
 class MessageBox:
-    def __init__(self, view, message):
+    def __init__(self, view, message, notes=None):
         self.message = message
+        self.notes = notes
         self.view = view
         self.width = 500
         self.height = 50
```

Comment:
We should also consider refactoring all of these classes to pydantic BaseModels, which will make debugging easier. It would look something like:

```py
from pydantic import BaseModel, PrivateAttr

class MessageBox(BaseModel):

    view: ?
    message: str
    notes: list[str] | None = None
    width: PrivateAttr[int] = 500

Reply:
Sure thing, I can do that in a future refactor